app/controllers/cats_controller.rb

```diff
@@ -1,30 +1,18 @@
 class CatsController < ApplicationController
 
-#         def index
-#             cats = Cat.all
-#             render json: cats
-#           end
-      
-#           def create
-#             # Create a new cat
-#             cat = Cat.create(cat_params)
-#             render json: cat
-#           end
-        
-#           # Handle strong parameters, so we are secure
-#           private
-#           def cat_params
-#             params.require(:cat).permit(:name, :age, :enjoys, :image)
-#           end
-      
-#         def update
-#         end
-      
-#         def destroy
-#         end
+def index
+    cats = Cat.all
+    render json: cats
+end
       
-#       end
-
+def create
+    # Create a new cat
+    cat = Cat.create(cat_params)
+    render json: cat
 end
 
-#   something
\ No newline at end of file
+private
+def cat_params
+    params.require(:cat).permit(:name, :age, :enjoys, :image)
+end   
```

Comment:
lines 3 through 17 should be indented one. That way it is nested inside the class.